### ISSUE TITLE
Function macro expansion

### ISSUE THREAD

**Issue by AsFoxy** [SkaceKamen/vscode-sqflint#14]
```cpp
#define MACRO(x) _x = x
MACRO(a);
```

Is incorrectly expanded as:
`_a = a;`

The correct expansion is:
`_x = a;`
